libexec/pyenv-init

```diff
@@ -97,10 +97,7 @@ esac
 
 completion="${root}/completions/pyenv.${shell}"
 if [ -r "$completion" ]; then
-  case "$shell" in
-  fish ) echo ". '$completion'" ;;
-  *    ) echo "source '$completion'" ;;
-  esac
+  echo "source '$completion'"
```

Comment:
This seems odd, but comes from rbenv/rbenv@8b0b51a (https://github.com/rbenv/rbenv/pull/873).